exporter/importables.go

```diff
@@ -560,18 +560,24 @@ var resourcesMap map[string]importable = map[string]importable{
 			return nil
 		},
 		Import: func(ic *importContext, r *resource) error {
-			if r.Name == "admins" || r.Name == "users" {
-				// admins & users are to be imported through "data block"
-				// TODO: it doesn't work, fix it...
-				r.Mode = "data"
-				r.Data.State().Set(&terraform.InstanceState{
-					ID: r.ID,
-					Attributes: map[string]string{
-						"display_name": r.Name,
-					},
-				})
-			} else if r.Data != nil {
-				r.Data.Set("force", true)
+			if r.Data != nil {
+				groupName := r.Data.Get("display_name").(string)
```

Comment:
could you please add unit tests for this change, so that the following commits won't break it?

Reply:
I can do that @nfx 

What should the unit test be looking for? An r.Data that **is nil**?
Before we decided to fix this bug, that was the original if condition.

Thanks